test/cmdlineTests.sh

```diff
@@ -71,6 +71,15 @@ function update_expectation {
     local newExpectation="${1}"
     local expectationFile="${2}"
 
+    if [[ $newExpectation == "" || $newExpectation -eq 0 ]]
```

Comment:
We only want this for the `exit` files. `0` might be valid content for some other files.
This should check if `$expectationFile` ends with `/exit`.